.gitignore

```diff
@@ -5,3 +5,7 @@
 /.cargo
 /.gomodcache
 /html
+/Infra.toml
+/*.pem
```

Comment:
This was the short-term workaround we agreed on until there's a standard process (in Makefile.toml) for making keys that would live in a known path (which would allow simplifying assumptions elsewhere).

Reply:
https://github.com/bottlerocket-os/bottlerocket/issues/999